docs/vNext/tutorials_fabric_existing.md

```diff
@@ -0,0 +1,724 @@
+---
+layout: vNext
+title:  "Setting up and Running a Performance Benchmark on an existing network"
+categories: 
+permalink: /vNext/fabric-tutorial/tutorials-fabric-existing/
+order: 
+---
+## Table of Contents
+{:.no_toc}
+
+- TOC
+{:toc}
+
+## Overview
+
+This tutorial takes you through performance testing a smart contract on a pre-existing Fabric network using Caliper. 
+
+This tutorial follows from the test-network [fabric tutorial](https://hlf.readthedocs.io/en/latest/test_network.html). You need to have installed and instantiated the fabcar smart contract (part two of the Fabric tutorial).
+
+
+## Step 1 - Install and Bind Caliper
+
+Globally install caliper CLI using the following terminal command:
+
+`npm install -g --only=prod @hyperledger/caliper-cli@0.4.0`
+
+Bind the SDK using the following terminal command:
+
+`caliper bind --caliper-bind-sut fabric:2.2.0 --caliper-bind-args=-g`
+
+## Step 2 - Setting Up Folders
+
+Caliper needs two configuration files:
+- The network configuration file that describes the network and provides the connection requirements. 
+- The benchmark file that defines the benchmark test rounds, through specification of a workload module and a series of options to drive the workload over a time interval. 
+
+Create a parent folder named **caliper-workspace** and then within this folder create two subfolders named **networks** and **benchmarks** respectively
```

Comment:
the tutorial assumes the caliper-workspace and fabric-samples are at the same dir level, such that `../fabric-samples` would be valid. Might be worth noting